tsconfig.json

```diff
@@ -14,7 +14,8 @@
     "noUnusedLocals": true,
     "pretty": true,
     "strict": true,
-    "target": "esnext"
+    "target": "esnext",
+    "types": []
```

Comment:
By default, TS loads all the types it can find in `node_modules/@types`, which can load undesired types (like `@types/node`) then leading to type conflicts (browser `setTimeout` vs node `setTimeout` for example).